app/services/local-storage.js

```diff
@@ -0,0 +1,48 @@
+import Ember from "ember";
+import config from "../config/environment";
+
+const LS = window.localStorage;
+const ENV = config.environment;
+const isProd = /^prod/.test(ENV);
+
+// --- Helpers
+
+function serialize(data) {
+  return JSON.stringify({ data });
+}
+
+function deserialize(str) {
+  try {
+    const obj = JSON.parse(str);
+    return obj && obj.data;
+  } catch (e) {
+    console.error(e);
+    return null;
+  }
+}
+
+function prefixKey(key) {
+  if (isProd) {
+    return key;
```

Comment:
Any reason why we don't prefix in production as well?

Reply:
I never prefixed it because if someone updates their app, it suddenly won't find the keys anymore. The prefix is mostly to prevent test data to be present when we run the app